src/olympia/versions/models.py

```diff
@@ -222,7 +222,16 @@ def __str__(self):
         return markupsafe.escape(self.version)
 
     @classmethod
-    def from_upload(cls, upload, addon, selected_apps, channel, parsed_data=None):
+    def from_upload(
+        cls,
+        upload,
+        addon,
+        channel,
+        *,
+        selected_apps=None,
+        compatiblity=None,
```

Comment:
typo `compatiblity` - which I think doesn't cause issues cause you seem to have made it everywhere you use that argument, but would be good to fix.